cmf-cli/Commands/new/BusinessCommand.cs

```diff
@@ -1,7 +1,7 @@
-using System;
```

Comment:
Missing this (a message is not being shown whenever a user tries to use the applicationVersion parameter without being an App):
![image](https://github.com/criticalmanufacturing/cli/assets/29643510/18d65b9c-6c55-4bdd-a738-c3c448961520)